frontend/views/components/menu/MenuTrigger.vue

```diff
@@ -1,13 +1,11 @@
 <template lang='pug'>
-button(
-  v-if='hideWhenActive ? !isActive : true'
-  :aria-pressed='isActive ? "true" : "false"'
+summary(
   data-test='menuTrigger'
-  :class='{ "is-active" : isActive }'
-  @click='handleClick'
+  :class='{ "button": true, "is-active" : isActive }'
```

Comment:
`summary(:class='{ "button": true, "is-active": isActive}')` here can be just `summary.button(:class='{ "is-active": isActive }')`